src/main/java/de/tum/in/www1/artemis/service/connectors/ContinuousIntegrationService.java

```diff
@@ -71,6 +71,25 @@
      */
     public Result onBuildCompleted(Participation participation);
 
+    /**
+     * Get the plan key of the finished build, the information of the build gets passed via the requestBody.
```

Comment:
Please add some more documentation here. Which requestBody, which REST call, maybe even link to the other Github Repo so that it's 100% clear

Reply:
I've added a link to the repository.